jaxp/src/com/sun/org/apache/xml/internal/serializer/ToStream.java

```diff
@@ -539,7 +539,7 @@ void setProp(String name, String val, boolean defaultVal) {
                 if (OutputPropertiesFactory.S_KEY_INDENT_AMOUNT.equals(name)) {
                     setIndentAmount(Integer.parseInt(val));
                 } else if (OutputKeys.INDENT.equals(name)) {
-                    boolean b = "yes".equals(val) ? true : false;
+                    boolean b = val.endsWith("yes") ? true : false;
```

Comment:
EndsWith is not very idiomatic for a boolean, maybe that requires an explaining comment?